collab-database/src/rows/row_meta.rs

```diff
@@ -95,16 +108,20 @@ impl RowMeta {
   pub(crate) fn empty() -> Self {
     Self {
       icon_url: None,
-      cover_url: None,
+      cover: None,
       is_document_empty: true,
       attachment_count: 0,
     }
   }
 
   pub(crate) fn from_map_ref<T: ReadTxn>(txn: &T, row_id: &Uuid, map_ref: &MapRef) -> Self {
+    let cover_data: String = map_ref
+      .get_with_txn(txn, &meta_id_from_row_id(row_id, RowMetaKey::CoverId))
+      .unwrap_or_default();
+
     Self {
       icon_url: map_ref.get_with_txn(txn, &meta_id_from_row_id(row_id, RowMetaKey::IconId)),
-      cover_url: map_ref.get_with_txn(txn, &meta_id_from_row_id(row_id, RowMetaKey::CoverId)),
```

Comment:
you needs to implement a custom json parser that parser icon url from old data. Otherwise, all old rowmeta's icon will be empty

Reply:
I've not touched the icon.

The cover is what I've changed, I don't believe we have ever used it, so there shouldn't be any data for covers in any rows.

As a precaution, how would I go about writing the parser to ensure data integrity?

Reply:
If we override any existing keys, we need to be cautious. Otherwise, the old data associated with those keys may still persist, potentially leading to inconsistent or unexpected results